sample/build.gradle.kts

```diff
@@ -0,0 +1,60 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+}
+
+apply {
+    from("$rootDir/config.gradle.kts")
+    from("$rootDir/ktlint.gradle")
+    from("$rootDir/detekt.gradle")
+}
+
+val buildConfig: Map<String, Any> by project
+val releaseConfig: Map<String, Any> by project
+val sonatype: Map<String, Any> by project
+
+android {
+    namespace = "com.infinum.libname.sample"
```

Comment:
Maybe a // TODO to change the "libname" to the actual name, same below and as we have in other places.